src/main/flight/pid.c

```diff
@@ -638,6 +637,17 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
 
             pidData[axis].D = pidCoefficient[axis].Kd * delta * tpaFactor;
 
+
+            const float pid_ff =
+                pidCoefficient[axis].Kd * dynCd * transition *
+                (currentPidSetpoint - previousPidSetpoint[axis]) * tpaFactor / dT;
+            if ((pidData[axis].P > 0) == (pid_ff > 0)) {
```

Comment:
`if (pidData[axis].P * pid_ff > 0)`

Reply:
Most definitely yes, it takes just 1 cycle.
Don't bother about such micro-optimizations, I'm measuring such stuff with a profiler.